libraries/AP_Notify/Buzzer.h

```diff
@@ -18,11 +18,16 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-
 #if CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
  # define BUZZER_PIN    32
+#elif CONFIG_HAL_BOARD == HAL_BOARD_LINUX
+ #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
+    #define BUZZER_PIN     44
+ #else
+    #define BUZZER_PIN     0
+ #endif
 #else
- # define BUZZER_PIN    0       // pin undefined on other boards
+    #define BUZZER_PIN     0
```

Comment:
Why remove the comment? There's no need to duplicate it to the other zero, just in this one is fine.